app/questionnaire_state/item.py

```diff
@@ -0,0 +1,28 @@
+
+
+class Item(object):
+    def __init__(self, id):
+        self.id = id
+        self.sections = []
+        self.children = None
+        self.is_valid = None
+        self.errors = None
+        self.warnings = None
+
+    def update_state(self, user_input):
+        for child in self.children:
+            child.update_state(user_input)
+
+    def get_answer(self, id):
```

Comment:
This will always return the first found answer - is there any likelyhood that this _won't_ be the answer? Should it check through all possibles and then return? Raise an exception? Return both values?